src/Round.js

```diff
@@ -1,42 +1,41 @@
 const Turn = require('../src/Turn');
-const game = require('../src/Game');
 const data = require('./data');
 const prototypeQuestions = data.prototypeData;
 const util = require('./util');
 const Deck = require('./Deck');
 
 class Round {
-  constructor(deck, turn) {
+  constructor(deck) {
     this.deck = deck.deck;
     this.turns = 0;
     this.incorrectGuesses = [];
     }
-    returnCurrentCard = (card) => {
-      return this.deck[0];
+    returnCurrentCard = () => {
+      return this.deck[0]
     }
-    takeTurn = (guess) => {     
+    takeTurn = (guess) => {   
       let currentCard = this.returnCurrentCard();
-      let newTurn = new Turn(guess, currentCard);
-      !newTurn.evaluateGuess() ? this.incorrectGuesses.unshift(currentCard.id) : null;
+      let turn = new Turn(guess, currentCard);
+      let result = turn.evaluateGuess();
+      !result ? this.incorrectGuesses.unshift(currentCard.id) : null;
       this.deck.shift();
       this.turns++;
-      return newTurn.giveFeedback();
+      return turn.giveFeedback()
```

Comment:
This is the final state I have decided on for the takeTurn method, it is readable, SRP compliant, and fully functional.